src/panels/lovelace/elements/types.ts

```diff
@@ -8,4 +8,5 @@ export interface LovelaceElementConfig {
 export interface LovelaceElement extends HTMLElement {
   hass?: HomeAssistant;
   setConfig(config: LovelaceElementConfig): void;
+  firstStyled?(): void;
```

Comment:
We can't just change the interface of all elements. Especially without documentation.

Reply:
I will gladly document once approved. It's a non breaking addition.